test/unit/trigger-method.spec.js

```diff
@@ -141,8 +141,8 @@ describe('trigger event and method name', function() {
       this.childView.$el.click();
     });
 
-    it('should fire the event method once', function() {
-      expect(this.onChildviewFooClickStub).to.have.been.calledOnce;
+    it('should fire the event method', function() {
+      expect(this.onChildviewFooClickStub).to.have.been.called;
```

Comment:
This is truly bazaar, for some reason the triggerMethod is called twice for what looks like a Mocha reason, but I have no idea why. Should be safe as it's the only instance of this